src/main/scala/org/hatdex/dex/api/models/Notice.scala

```diff
@@ -11,6 +11,7 @@ package org.hatdex.dex.api.models
 
 import org.joda.time.DateTime
 import play.api.libs.json.Json
+import play.api.libs.json.{ OWrites, Reads }
```

Comment:
What is the `OWrites` ?? 🤯

Reply:
https://www.playframework.com/documentation/2.6.13/api/scala/index.html#play.api.libs.json.OWrites

Converts Objects to JsValues.